c/zss.c

```diff
@@ -560,6 +567,41 @@ static void installWebPluginDefintionsService(WebPluginListElt *webPlugins, Http
   zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG2, "end %s\n", __FUNCTION__);
 }
 
+static void parseLoggingData(HttpServer *server, ShortLivedHeap *slh, const char *filename) {
+
+  char errorBuffer[512] = { 0 };
+  JsonObject *jsonObject = NULL;
+  
+  Json *json = jsonParseFile(slh, filename, errorBuffer, sizeof(errorBuffer));
+  if (json) {
+    if (jsonIsObject(json)) {
+      jsonObject = jsonAsObject(json);
+    }
+    else {
+      zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_WARNING, "error while parsing %s: %s\n", errorBuffer);
+    }
+    JsonObject *logLevels = jsonObjectGetObject(jsonObject, "logLevels");
+    JsonProperty *property = jsonObjectGetFirstProperty(logLevels);
+    int i = 1;
+    while(property != NULL) {
+      char *key = jsonPropertyGetKey(property); 
```

Comment:
Can this be ```NULL```? If yes, the next call will crash.

Reply:
It can't be NULL. The server will not start if invalid JSON is present in the zluxserver.json file. Also, if the property is null, this code won't execute. In this case, it won't be null, so I don't think it needs to be checked.